resources/rpm/project.spec.erb

```diff
@@ -118,10 +118,10 @@ done
 %pre
 # Save state so we know later if this is an upgrade or an install
 mkdir -p %{_localstatedir}/lib/rpm-state/%{name}
-if [ $1 -eq 1 ] ; then
+if [ "$1" -eq 1 ] ; then
```

Comment:
@shrug and I had some back and forth about how to do this, and it worked out to:

```
if you're comparing numbers in bash, always use -eq
if you're comparing strings in bash, always use =
that should work F*** EVERYWHERE
```

As for comparing a quoted variable to an unquoted value, the bottom line is "quote the interpolated variable regardless of type" because if something goes sideways, `[ "" -eq 1 ]` is valid, whereas `[ -eq 1]` is not.